src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,115 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private Node<K, V>[] table;
+    private int size = 0;
+    private Node<K, V> nullKeyNode = null;
+
+    public MyHashMap() {
+        table = new Node[INITIAL_CAPACITY];
```

Comment:
Remember about the diamond operator. You should use it like new Node<K,V>[...] to avoid unchecked assignment warnings.